docs/client/full-api/commandline.html

```diff
@@ -40,6 +40,20 @@ <h3 id="meteorrun">meteor run</h3>
 Run `meteor help run` to see the full list of options.
 
 
+<h3 id="meteordebug">meteor debug</h3>
+
+Run the project, but suspend the server process for debugging.
+
+The server process will be suspended just before the first statement of
+server code that would normally execute. In order to continue execution of
+server code, use either the web-based Node Inspector or the command-line
+debugger (further instructions will be printed in the console).
+
+The easiest way to set breakpoints is to use the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger">`debugger` keyword</a>.
+
+The options for `meteor debug` are identical to those for `meteor run`, with the addition of `--debug-port`. Run `meteor help debug` for full details.
```

Comment:
Maybe worth saying that you can get the same functionality by adding the `--debug-port` argument to some other commands such as `test-packages`.